python/paddle/fluid/dygraph/dygraph_to_static/utils.py

```diff
@@ -17,41 +17,22 @@
 import inspect
 import gast
 import astor
-import atexit
-import os
-import tempfile
-import six
-import imp
+from .static_analysis import is_dygraph_api
```

Comment:
I think it is better to put "is_dygraph_api" into utils.py, not static_analysis.py because it is like a utility everyone can use. Also put other utility functions into utils.py?

Reply:
Thanks. I will modify it